scripts/dotnet-cli-build/PublishTargets.cs

```diff
@@ -24,6 +26,8 @@ public static class PublishTargets
 
         private static string SharedFrameworkNugetVersion { get; set; }
 
+        private static List<string> _latestBlobs;
```

Comment:
I think the general approach is to put properties into the BuildTargetContext instead of having static variables.

But looking deeper - it looks like you can just return the List<string> as an out parameter of `CheckIfAllBuildsHavePublished` and remove this static variable.